src/libmscore/chordlist.cpp

```diff
@@ -872,14 +872,14 @@ bool ParsedChord::parse(const QString& s, const ChordList* cl, bool syntaxOnly,
             ++i;
         }
         // get second token - a number <= 13
```

Comment:
Actually this code accepts any number than starts with 10, 11, 12, or 13, such as 12345. I did not attempt to fix it in case it breaks some usage.